ethereum/test/test_eth_app.js

```diff
@@ -81,6 +120,7 @@ contract("EthApp", function (accounts) {
         // Send to a substrate recipient to load contract with unlockable ETH
         await this.ethApp.sendETH(
           substrateRecipient,
+          true,
```

Comment:
Likewise?

Reply:
refactored tests, have separate tests for the apps and for channels, will push soon.